src/Workspaces/CSharp/Portable/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -1142,5 +1143,29 @@ public static ConditionalAccessExpressionSyntax GetInnerMostConditionalAccessExp
 
             return false;
         }
+
+        public static SyntaxNode MoveNonIndentationTriviaTo(
```

Comment:
I admit I'm not a fan of this as an extension method, or might prefer it in the form .WithNonIndentationTrivia() with the parameters swapped...it's just confusing which direction it's going.